### PR TITLE
Setting docker command in docker builder

### LINKED ISSUE #131: Docker builder should set container command
Instead of relying on the pod spec. This will help with debugging the generated container locally and also for submitting to additional environments.

### PR DESCRIPTION
**Pull request by karthikv2k**
Fixes #131

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/132)

### CODE REVIEW

File: fairing/builders/dockerfile.py [hunk 1]
```diff
@@ -14,7 +15,8 @@ def write_dockerfile(
         "ENV FAIRING_RUNTIME 1",
         "COPY {PATH_PREFIX} {PATH_PREFIX}".format(PATH_PREFIX=path_prefix),
         "RUN if [ -e requirements.txt ];" +
-        "then pip install --no-cache -r requirements.txt; fi"
+        "then pip install --no-cache -r requirements.txt; fi",
+        "CMD {}".format(" ".join(docker_command))
```
[the remark this answers is not in the document]
Comment (karthikv2k) [reply fairing/builders/dockerfile.py]: Is there a case where we don't want to set a docker command?

[the remark this answers is not in the document]
Comment (karthikv2k) [reply fairing/builders/dockerfile.py]: made docker command optional.

File: fairing/builders/docker/docker.py [hunk 2]
```diff
@@ -35,7 +35,12 @@ def build(self):
             self.publish()
 
     def _build(self):
+        docker_command = self.preprocessor.get_command()
+        log.warning("Docker command: {}".format(docker_command))
```
Comment (rpasricha) [inline fairing/builders/docker/docker.py]: Should this be logger.warning?

Comment (karthikv2k) [reply fairing/builders/docker/docker.py]: corrected. Thanks.

**Pull request merged**